### PR TITLE
Murders The Devil from The Bible in the form of a bee

### PR DESCRIPTION
**Pull request by Skies-Of-Blue**
## About The Pull Request
Removes chem bees from the jungle and beach spawn pools, replacing them with normal bees.

## Why It's Good For The Game
Walking off of a ship onto one of the relatively "peaceful" planets, only to be loaded up with admin chemicals was a pretty awful experience. 

We inherited these little guys from Voidcrew, honestly that should be enough reason to remove them.

## Changelog

:cl:
balance: planetary chem bees have been replaced with the garden variety
/:cl:

### CODE REVIEW

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -161,6 +163,9 @@
 				new /obj/item/clothing/suit/armor/ascetic(src)
 			if(29)
 				new /obj/item/kitchen/knife/envy(src)
+			if(30)
+				new /mob/living/simple_animal/hostile/poison/bees/toxin(src)
```
Comment (Zevotech) [inline code/modules/mining/lavaland/necropolis_chests.dm]: as funny as this is, you probably shouldn't do it.

Comment (Skies-Of-Blue) [reply code/modules/mining/lavaland/necropolis_chests.dm]: Ah, but it was so worth a try!

Comment (Zevotech) [reply code/modules/mining/lavaland/necropolis_chests.dm]: It would be so so so so SO funny. But not this item pool. Maybe something else, later.

Comment (Skies-Of-Blue) [reply code/modules/mining/lavaland/necropolis_chests.dm]: As a fun fact, they already HAVE existed in a different item pool. Went back and forth on removing them there, but based on the fact that I haven't heard a single player discover it yet, I assume it's fine.

**Review by MarkSuckerberg — APPROVED** [review]
I'll never forget the round I was swarmed by bungotoxin bees

no matter how hard I try

**Comment by Addust** [comment]
on behalf of the voidcrew community:

image

**Pull request merged**